shared-data/contrib/hints/hints/organize-sidebar.html

```diff
@@ -0,0 +1,31 @@
+{% extends "layouts/" + render_mode + "-tall.html" %}
+{% block title %}{{_("Organizing Your Sidebar")}}{% endblock %}
+{% block content %}
+<div class="content-normal" style="max-width: 46em;">
+  <h1><span class="icon-tag"></span>
+    {{_("Organizing Your Sidebar")}}
+  </h1>
+  <p>
+    {{_("Mailpile lets you organize your mailbox the way you want it be.")}}<br>
+    {{_("You can start customizing your experince by rearranging the sidebar on the left.")}}
+  </p>
+  <ul class="list" style="margin-left: 2em; list-style: disc">
+    <li>{{_("In the bottom left, click ")}}<b>{{_("Organize.")}}</b></li>
```

Comment:
We try to avoid breaking sentences into fragments, since different languages prefer a different order. In cases like this though, because of the formatting, it can be hard to avoid. Generally what we've done is follow this pattern:

In the bottom left, click: Organize.

In the code that would look like this:

    {{_("In the bottom left, click:")}} <b>{{_("Organize")}}.</b>

Note the differences in punctuation and spacing - I put the : inside the string, as a hint to the translator that something will come after, but took the white space out (since translators might accidentally remove it). However, I've removed the period from the translatable word "Organize", as that same word may be used/translated elsewhere without the punctuation being appropriate.